mobile_verifier/src/heartbeats.rs

```diff
@@ -287,3 +287,14 @@ async fn validate_heartbeat(
 
     Ok((cell_type, proto::HeartbeatValidity::Valid))
 }
+
+pub async fn clear_heartbeats(
+    tx: &mut sqlx::Transaction<'_, sqlx::Postgres>,
+    reward_period: &Range<DateTime<Utc>>,
```

Comment:
I would recommend not taking a period here since it is unnecessary, have this be `reward_start: DateTime<Utc>` (same for clear_speedtests)

Reply:
addressed here: ae8fb0607f4b32f6881a7ce46e00551de941f410